tests/test_masses.py

```diff
@@ -0,0 +1,99 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+from astropy.cosmology import Planck18
+
+from GWFish.modules.detection import Detector, Network
+from GWFish.modules.fishermatrix import compute_network_errors
+
+def from_m1_m2_to_mChirp_q(m1, m2):
+    """
+    Compute the transformation from m1, m2 to mChirp, q
+    """
+    mChirp = (m1 * m2)**(3/5) / (m1 + m2)**(1/5)
+    q = m2 / m1
+    return mChirp, q
+
+def test_snr_different_mass_inputs():
+
+    detectors = ['ET']
+    network = Network(detector_ids = detectors, detection_SNR = (0., 8.))
+    waveform_model = 'IMRPhenomD_NRTidalv2'
+
+    z = np.array([0.00980])
+
+    parameters_source = {
```

Comment:
Really nice! 

I would just propose a little cleanup: you can define a dictionary of `extra params` (ra, dec...) just once, and then each time combine them like

```
params_det = {masses_det} | extra_params
```
and similarly for the others, so you only change the masses

Reply:
and it makes the function more readable

Reply:
Nice, on it!